1.9/administration/metrics/index.md

```diff
@@ -3,4 +3,142 @@ post_title: Metrics
 menu_order: 3.5
 ---
 
+The [metrics component](https://github.com/dcos/dcos-metrics) provides operational insight to your DC/OS cluster, providing discrete metrics about your applications and deployments. This can include charts, dashboards, and alerts based on cluster, node, container, and application-level statistics. The metrics component is natively integrated with DC/OS version 1.9 and later. No additional setup is required.  
```

Comment:
`The metrics component provides metrics from DC/OS cluster hosts, containers running on those hosts, and from applications running on DC/OS which choose to send statsd metrics to our Mesos Metrics Module.` 1st sentence change.